packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -91,3 +91,14 @@ export const borderAndBoxShadowStyle = (
     }
   };
 };
+
+/**
+ * @private
+ */
+export const errorBarStyle = (theme: Theme): IStyle => {
+  return {
+    background: '#FFF4CE',
```

Comment:
Could you look at the ErrorBar component styling to see how it uses colors.
This might not work when a dark theme is used.

Reply:
<img width="1326" alt="Screen Shot 2022-02-25 at 4 09 26 PM" src="https://user-images.githubusercontent.com/97124699/155824393-61abb0d5-fa8f-45d8-98f0-7c797a0a68c2.png">
The existing error bar in storybook has same styling

Reply:
We should still use theme here in case Contoso has set a custom color for warning messages.
This is under `theme.semanticColors.warningBackground` (or better, use a Fluent MessageBar with `messageBarType={MessageBarType.Warning}`